### DISCUSSION TITLE
Welcome to sonar-cryptography Discussions!

### DISCUSSION THREAD

**Discussion by n1ckl0sk0rtge** [IBM/sonar-cryptography#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about  relating to the development of the plugin.
  * Provide feedback on your experience using the sonar cryptography plugin.
  * Upvote discussions that you find useful.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## Get Started

Before opening a new discussion, check if something similar has already been discussed before and avoid duplication where possible! If your comment/query has not already been opened feel free to open your own.

When you open a discussion please choose the appropriate category and it really helps to use labels as well!

Remember to comment and upvote 👍 on existing discussions too!

### Discussion Categories

You can open a discussion in any of the following areas:

- 💡 Ideas : If you have suggestions for new features or things you would like to see in the plugin this is the place to share them and get feedback from the community!
- 🛣 Releases and Roadmap: Ask questions about past and upcoming releases, and future plans for the Sonar Cryptography Plugin.
- 📚 Documentation: Provide feedback on the documentation.
- 💬 General Feedback: Provide feedback on your experience using the plugin (anything that doesn't fit into the other categories)

**Comment by Ayush-Patel-56** [comment]
Hi @n1ckl0sk0rtge, I just wrote a blog on [Medium](https://medium.com/@ayushpatel5615/how-sonar-cryptography-detects-cryptographic-assets-and-what-it-took-to-understand-it-0bb65592a93d), love to hear your thoughts. Thanks!
